### PR TITLE
Open CFP

### PR DESCRIPTION
**Pull request by plaindocs**
- [x] pending CFP going live on Pretalx

### CODE REVIEW

**Comment by plaindocs** [comment]
Good catch on the timezone.

File: docs/conf/portland/2021/news/cfp-opens.rst [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+You can read our full `Call for Proposals <https://www.writethedocs.org/conf/portland/{{year}}/cfp/>`__ on the website.
+
+The Call for Proposals will be open until **Midnight PST {{cfp.ends}}**.
+
+Tickets
+-------
+
+Tickets will officially go on sale in **January 2020**.
+
+In the meantime, you can see the ticket prices on our `Tickets <https://www.writethedocs.org/conf/portland/{{year}}/tickets/>`_ page.
```
Comment (ericholscher) [inline docs/conf/portland/2021/news/cfp-opens.rst:29]: We haven't updated ticket prices, have we? I'll adjust them now.

Comment (ericholscher) [reply docs/conf/portland/2021/news/cfp-opens.rst:29]: Done.

**Review by ericholscher — APPROVED** [review]
Think this is ready to go 👍

**Pull request merged**